core/src/main/java/io/opentelemetry/android/export/FilteringSpanExporterBuilder.java

```diff
@@ -16,6 +19,19 @@ public final class FilteringSpanExporterBuilder {
 
     private final SpanExporter delegate;
     private Predicate<SpanData> predicate = x -> false;
+    private final Interceptor<SpanData> interceptor =
+            new Interceptor<SpanData>() {
+
+                @Nullable
+                @Override
+                public SpanData intercept(SpanData item) {
+                    if (predicate.test(item)) {
+                        return null;
+                    } else {
+                        return item;
+                    }
```

Comment:
Prefer to remove the redundant `else`.
```suggestion
                    if (predicate.test(item)) {
                        return null;
                    } 
                    return item;                   
```